jenkins/validate-artifacts/validate-artifacts.jenkinsfile

```diff
@@ -65,6 +76,19 @@ pipeline {
             trim: true,
             defaultValue: "using-staging-artifact-only"
         )
+        string(
+            name: 'PROJECTS',
+            description: 'Enter opensearch or opensearch-dashboards',
+            trim: true,
+            defaultValue: "opensearch"
+        )
+        string(
+            name: 'ARTIFACT_TYPE',
+            description: 'Determine the artifact type among staging or production ',
```

Comment:
Is local an option too? Should this be a `CHOICE` instead of string parameter?

Reply:
Local is not an option, it's automatically detected in python code based on the url format. User can just provide OS local path and OSD local path in this case. In this case we just copy the local artifacts from their respective path to temporary folder instead of downloading them.